ftests/nuxeo-server-tests/pom.xml

```diff
@@ -80,6 +80,14 @@
       <artifactId>nuxeo-platform-oauth1</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!-- excluded from chemistry-opencmis-commons-impl for groupId & version issues -->
+    <dependency>
+      <groupId>com.fasterxml.woodstox</groupId>
+      <artifactId>woodstox-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I'm not sure this is really needed, the dependency added to `nuxeo-opencmis-impl` should be enough.

It should be tested again without it.

Reply:
The list of failing tests was not exactly the same for Master, there is one extra test that was failing because it was missing the dependency:
master https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob/146/
10.10 https://qa2.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob-10.10/144/

Reply:
The server is returning `500`, I don't see why adding a test dependency to the server ftests would fix the issue... 🤔 

Is it possible to relaunch it w/o the dep so that we can see the server logs? :)

Reply:
Sure
I hope I can pass through the randoms !

Reply:
(but about the 500 error, since it's a test on a server which fails, I'm not surprized to have a 500 though)

Reply:
The error is server side, so it's related to jars *in* the server. Here you are adding a dependency to a functional tests module, it won't end up *in* the server.

Adding the dependency to the `nuxeo-opencmis-impl` module should be enough as this module (and its dependencies) will be deployed *in* the server.

Reply:
🟢  https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob/153/